data/helpers.d/print

```diff
@@ -54,6 +54,24 @@ ynh_print_log () {
   echo -e "${1}"
 }
 
+# Check if the current terminal supports colors
+#
+# [internal]
+#
+are_colors_supported () {
```

Comment:
Can't we optimize this a bit and check this once, globally, rather than every time we run a `ynh_print_*` ?

Reply:
Not sure we would save any time.
That would means runs that helper at the beginning of any script. Or even when the file is sourced. Do we want to do that ?